net-p2p/retroshare/retroshare-0.6.7.ebuild

```diff
@@ -0,0 +1,133 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic desktop qmake-utils xdg
+
+DESCRIPTION="F2F, private and secure file sharing application"
+HOMEPAGE="https://retroshare.cc"
+SRC_URI="http://download.opensuse.org/repositories/network:/retroshare/Debian_Testing/retroshare-common_${PV}.orig.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/RetroShare"
+# REVIEW(setan): we could build from the link belowe to be closer to upstream
+# but this requires git modules
+# https://github.com/RetroShare/RetroShare/archive/refs/tags/{PV}.tar.gz
+
+LICENSE="AGPL-3 Apache-2.0 CC-BY-SA-4.0 GPL-2 GPL-3 LGPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="keyring cli +gui +jsonapi libupnp +miniupnp +service +sqlcipher plugins"
+
+REQUIRED_USE="
+	|| ( gui service )
+	?? ( libupnp miniupnp )
+	service? ( || ( cli jsonapi ) )"
+
+RDEPEND="
+	app-arch/bzip2
+	dev-libs/openssl:0=
+	>=dev-libs/rapidjson-1.1.0
+	sys-libs/zlib
+	keyring? ( app-crypt/libsecret )
+	gui? (
+		dev-qt/qtcore:5
+		dev-qt/qtmultimedia:5
+		dev-qt/qtnetwork:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtscript:5
+		dev-qt/qtxml:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtx11extras:5
+		x11-libs/libX11
+		x11-libs/libXScrnSaver
+	)
+	libupnp? ( net-libs/libupnp:= )
+	miniupnp? ( net-libs/miniupnpc:= )
+	service? ( dev-qt/qtcore:5 )
+	sqlcipher? ( dev-db/sqlcipher )
+	!sqlcipher? ( dev-db/sqlite:3 )
+	plugins? ( media-libs/speex )"
+
+DEPEND="${RDEPEND}
+	dev-qt/qtcore:5
```

Comment:
similar thing happens here, qtcore is needed only for retroshare-gui at run time so this line should be removed

Reply:
It seems there is more about those variables I suggest to read https://devmanual.gentoo.org/ebuild-writing/variables/index.html documentation section about those.

Reply:
I have read this
https://devmanual.gentoo.org/general-concepts/dependencies/
But not the link you posted. Thanks.

Reply:
Is the qtcore in a good place now?

Reply:
yeah qtcore dependency seems good now